loanmanagement/settings.py

```diff
@@ -59,7 +48,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': ['templates'],
```

Comment:
We will use templates?

Reply:
I added a squad screen to the django index